addon/components/paper-contact-chips.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: 'md-contact-chips',
+  classNames: ['md-default-theme'],
+
+  actions: {
+    removeItem(item) {
```

Comment:
Can an action closure be used to here avoid having to ferry up remove and add actions?